mmv1/products/networkservices/TlsRoute.yaml

```diff
@@ -0,0 +1,159 @@
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  TlsRoute defines how traffic should be routed based on SNI and other matching L3 attributes.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1beta1/projects.locations.tlsRoutes'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+import_format: ['projects/{{project}}/locations/global/tlsRoutes/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_tls_route_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-tls-route'
+      backend_service_name: "my-backend-service"
+      http_health_check_name: "backend-service-health-check"
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_tls_route_mesh_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-tls-route'
+      backend_service_name: "my-backend-service"
+      http_health_check_name: "backend-service-health-check"
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_tls_route_gateway_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-tls-route'
+      backend_service_name: "my-backend-service"
+      http_health_check_name: "backend-service-health-check"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    url_param_only: true
+    immutable: true
+    description: |
+      Name of the TlsRoute resource.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'selfLink'
+    description: |
+      Server-defined URL of this resource.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: |
+      Time the TlsRoute was created in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: |
+      Time the TlsRoute was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      A free-text description of the resource. Max length 1024 characters.
+  - !ruby/object:Api::Type::Array
+    name: meshes
+    item_type: Api::Type::String
+    description: |
+          List of meshes this TlsRoute is attached to, as one of the routing rules to route the requests served by the mesh.  
```

Comment:
Could you please update this description to include the required format mentioned in the API reference?

Reply:
Added